#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_missing_includes.dpatch by Yavor Doganov <yavor@gnu.org>
##
## DP: Fixes to build with recent GNUstep Base and GNUstep Make.

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' projectmanager.app-0.2~/ProjectManager-0.2/Controller.m projectmanager.app-0.2/ProjectManager-0.2/Controller.m
--- projectmanager.app-0.2~/ProjectManager-0.2/Controller.m	2010-05-21 12:25:06.000000000 +0300
+++ projectmanager.app-0.2/ProjectManager-0.2/Controller.m	2010-05-21 12:25:18.000000000 +0300
@@ -23,21 +23,7 @@
 
 #import "Controller.h"
 
-#import <Foundation/NSBundle.h>
-#import <Foundation/NSSortDescriptor.h>
-#import <Foundation/NSString.h>
-#import <Foundation/NSFileManager.h>
-#import <Foundation/NSUserDefaults.h>
-#import <Foundation/NSNotification.h>
-
-#import <AppKit/NSOpenPanel.h>
-#import <AppKit/NSView.h>
-#import <AppKit/NSNibLoading.h>
-#import <AppKit/NSTextField.h>
-#import <AppKit/NSDocumentController.h>
-#import <AppKit/NSMenu.h>
-#import <AppKit/NSMenuItem.h>
-#import <AppKit/NSApplication.h>
+#import <AppKit/AppKit.h>
 
 #import "Preferences.h"
 #import "ProjectCreator.h"
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' projectmanager.app-0.2~/ProjectManager-0.2/EditorRulerView.m projectmanager.app-0.2/ProjectManager-0.2/EditorRulerView.m
--- projectmanager.app-0.2~/ProjectManager-0.2/EditorRulerView.m	2010-05-21 12:25:06.000000000 +0300
+++ projectmanager.app-0.2/ProjectManager-0.2/EditorRulerView.m	2010-05-21 12:25:18.000000000 +0300
@@ -23,17 +23,7 @@
 
 #import "EditorRulerView.h"
 
-#import <AppKit/NSColor.h>
-#import <AppKit/NSEvent.h>
-#import <AppKit/NSFont.h>
-#import <AppKit/NSGraphics.h>
-#import <AppKit/NSScrollView.h>
-#import <AppKit/NSStringDrawing.h>
-#import <AppKit/NSWindow.h>
-#import <AppKit/PSOperators.h>
-
-#import <Foundation/NSKeyedArchiver.h>
-#import <Foundation/NSUserDefaults.h>
+#import <AppKit/AppKit.h>
 
 #import "EditorTextView.h"
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' projectmanager.app-0.2~/ProjectManager-0.2/Preferences.m projectmanager.app-0.2/ProjectManager-0.2/Preferences.m
--- projectmanager.app-0.2~/ProjectManager-0.2/Preferences.m	2010-05-21 12:25:06.000000000 +0300
+++ projectmanager.app-0.2/ProjectManager-0.2/Preferences.m	2010-05-21 12:25:18.000000000 +0300
@@ -22,15 +22,7 @@
 
 #import "Preferences.h"
 
-#import <Foundation/NSBundle.h>
-#import <Foundation/NSString.h>
-#import <Foundation/NSArray.h>
-#import <Foundation/NSDictionary.h>
-
-#import <AppKit/NSNibLoading.h>
-#import <AppKit/NSBox.h>
-#import <AppKit/NSPopUpButton.h>
-#import <AppKit/NSPanel.h>
+#import <AppKit/AppKit.h>
 
 #import "PreferencesModule.h"
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' projectmanager.app-0.2~/ProjectManager-0.2/ProjectCreator.m projectmanager.app-0.2/ProjectManager-0.2/ProjectCreator.m
--- projectmanager.app-0.2~/ProjectManager-0.2/ProjectCreator.m	2010-05-21 12:25:17.000000000 +0300
+++ projectmanager.app-0.2/ProjectManager-0.2/ProjectCreator.m	2010-05-21 12:25:18.000000000 +0300
@@ -20,27 +20,7 @@
 
 #import "ProjectCreator.h"
 
-#import <Foundation/NSBundle.h>
-#import <Foundation/NSSortDescriptor.h>
-#import <Foundation/NSString.h>
-#import <Foundation/NSFileManager.h>
-#import <Foundation/NSUserDefaults.h>
-#import <Foundation/NSNotification.h>
-#import <Foundation/NSError.h>
-
-#import <AppKit/NSOpenPanel.h>
-#import <AppKit/NSView.h>
-#import <AppKit/NSNibLoading.h>
-#import <AppKit/NSTextField.h>
-#import <AppKit/NSDocumentController.h>
-#import <AppKit/NSMenu.h>
-#import <AppKit/NSMenuItem.h>
-#import <AppKit/NSApplication.h>
-#import <AppKit/NSForm.h>
-#import <AppKit/NSFormCell.h>
-#import <AppKit/NSScrollView.h>
-#import <AppKit/NSButton.h>
-#import <AppKit/NSOutlineView.h>
+#import <AppKit/AppKit.h>
 
 #import <../WizardKit-0.1/WizardKit.h>
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' projectmanager.app-0.2~/ProjectManager-0.2/ProjectDocument.m projectmanager.app-0.2/ProjectManager-0.2/ProjectDocument.m
--- projectmanager.app-0.2~/ProjectManager-0.2/ProjectDocument.m	2010-05-21 12:25:06.000000000 +0300
+++ projectmanager.app-0.2/ProjectManager-0.2/ProjectDocument.m	2010-05-21 12:25:18.000000000 +0300
@@ -23,18 +23,7 @@
 
 #import "ProjectDocument.h"
 
-#import <Foundation/NSString.h>
-#import <Foundation/NSArray.h>
-#import <Foundation/NSDictionary.h>
-#import <Foundation/NSUserDefaults.h>
-#import <Foundation/NSProcessInfo.h>
-
-#import <AppKit/NSWorkspace.h>
-#import <AppKit/NSDocumentController.h>
-#import <AppKit/NSImage.h>
-#import <AppKit/NSPanel.h>
-#import <AppKit/NSOpenPanel.h>
-#import <AppKit/NSButton.h>
+#import <AppKit/AppKit.h>
 
 #import "SourceEditorDocument.h"
 #import "ProjectWindowController.h"
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' projectmanager.app-0.2~/ProjectManager-0.2/ProjectModules/FileManager/ProjectImageView.m projectmanager.app-0.2/ProjectManager-0.2/ProjectModules/FileManager/ProjectImageView.m
--- projectmanager.app-0.2~/ProjectManager-0.2/ProjectModules/FileManager/ProjectImageView.m	2010-05-21 12:25:06.000000000 +0300
+++ projectmanager.app-0.2/ProjectManager-0.2/ProjectModules/FileManager/ProjectImageView.m	2010-05-21 12:25:18.000000000 +0300
@@ -23,13 +23,7 @@
 
 #import "ProjectImageView.h"
 
-#import <Foundation/NSArray.h>
-
-#import <AppKit/NSEvent.h>
-#import <AppKit/NSPasteboard.h>
-#import <AppKit/NSWindow.h>
-#import <AppKit/NSDragging.h>
-#import <AppKit/NSImage.h>
+#import <AppKit/AppKit.h>
 
 #import "FileManager.h"
 #import "FileManagerDelegate.h"
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' projectmanager.app-0.2~/ProjectManager-0.2/ProjectModules/GNUstepAppLauncher/GNUstepAppLauncher.m projectmanager.app-0.2/ProjectManager-0.2/ProjectModules/GNUstepAppLauncher/GNUstepAppLauncher.m
--- projectmanager.app-0.2~/ProjectManager-0.2/ProjectModules/GNUstepAppLauncher/GNUstepAppLauncher.m	2010-05-21 12:25:06.000000000 +0300
+++ projectmanager.app-0.2/ProjectManager-0.2/ProjectModules/GNUstepAppLauncher/GNUstepAppLauncher.m	2010-05-21 12:25:18.000000000 +0300
@@ -23,30 +23,7 @@
 
 #import "GNUstepAppLauncher.h"
 
-#import <Foundation/NSBundle.h>
-#import <Foundation/NSException.h>
-#import <Foundation/NSFileHandle.h>
-#import <Foundation/NSNotification.h>
-#import <Foundation/NSProcessInfo.h>
-#import <Foundation/NSString.h>
-#import <Foundation/NSTask.h>
-
-#import <AppKit/NSColor.h>
-#import <AppKit/NSFont.h>
-#import <AppKit/NSFormCell.h>
-#import <AppKit/NSForm.h>
-#import <AppKit/NSImage.h>
-#import <AppKit/NSMenuItem.h>
-#import <AppKit/NSNibLoading.h>
-#import <AppKit/NSOpenPanel.h>
-#import <AppKit/NSPopUpButton.h>
-#import <AppKit/NSTableColumn.h>
-#import <AppKit/NSTableView.h>
-#import <AppKit/NSTextStorage.h>
-#import <AppKit/NSTextView.h>
-#import <AppKit/NSWindow+Toolbar.h>
-#import <AppKit/NSToolbar.h>
-#import <AppKit/NSToolbarItem.h>
+#import <AppKit/AppKit.h>
 
 #import "GNUstepAppLauncherDelegate.h"
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' projectmanager.app-0.2~/ProjectManager-0.2/ProjectModules/MakeBuilder/MakeBuilder.m projectmanager.app-0.2/ProjectManager-0.2/ProjectModules/MakeBuilder/MakeBuilder.m
--- projectmanager.app-0.2~/ProjectManager-0.2/ProjectModules/MakeBuilder/MakeBuilder.m	2010-05-21 12:25:06.000000000 +0300
+++ projectmanager.app-0.2/ProjectManager-0.2/ProjectModules/MakeBuilder/MakeBuilder.m	2010-05-21 12:25:18.000000000 +0300
@@ -22,31 +22,7 @@
 
 #import "MakeBuilder.h"
 
-#import <Foundation/NSArray.h>
-#import <Foundation/NSException.h>
-#import <Foundation/NSFileHandle.h>
-#import <Foundation/NSNotification.h>
-#import <Foundation/NSString.h>
-#import <Foundation/NSTask.h>
-#import <Foundation/NSUserDefaults.h>
-
-#import <AppKit/NSButton.h>
-#import <AppKit/NSCell.h>
-#import <AppKit/NSDocumentController.h>
-#import <AppKit/NSImage.h>
-#import <AppKit/NSMatrix.h>
-#import <AppKit/NSMenuItem.h>
-#import <AppKit/NSNibLoading.h>
-#import <AppKit/NSPanel.h>
-#import <AppKit/NSPopUpButton.h>
-#import <AppKit/NSScroller.h>
-#import <AppKit/NSScrollView.h>
-#import <AppKit/NSTableColumn.h>
-#import <AppKit/NSTableView.h>
-#import <AppKit/NSToolbar.h>
-#import <AppKit/NSToolbarItem.h>
-#import <AppKit/NSView.h>
-#import <AppKit/NSWindow+Toolbar.h>
+#import <AppKit/AppKit.h>
 
 #import "MakeBuilderDelegate.h"
 #import "../../ProjectDocument.h"
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' projectmanager.app-0.2~/ProjectManager-0.2/ProjectModules/SubprojectsManager/SubprojectsManager.m projectmanager.app-0.2/ProjectManager-0.2/ProjectModules/SubprojectsManager/SubprojectsManager.m
--- projectmanager.app-0.2~/ProjectManager-0.2/ProjectModules/SubprojectsManager/SubprojectsManager.m	2006-07-09 17:11:42.000000000 +0300
+++ projectmanager.app-0.2/ProjectManager-0.2/ProjectModules/SubprojectsManager/SubprojectsManager.m	2010-05-21 12:27:49.000000000 +0300
@@ -23,22 +23,7 @@
 
 #import "SubprojectsManager.h"
 
-#import <Foundation/NSArray.h>
-#import <Foundation/NSDictionary.h>
-#import <Foundation/NSError.h>
-#import <Foundation/NSFileManager.h>
-#import <Foundation/NSNotification.h>
-#import <Foundation/NSSortDescriptor.h>
-
-#import <AppKit/NSDocumentController.h>
-#import <AppKit/NSNibLoading.h>
-#import <AppKit/NSOpenPanel.h>
-#import <AppKit/NSPanel.h>
-#import <AppKit/NSTableColumn.h>
-#import <AppKit/NSOutlineView.h>
-#import <AppKit/NSToolbarItem.h>
-#import <AppKit/NSView.h>
-#import <AppKit/NSWindow.h>
+#import <AppKit/AppKit.h>
 
 #import "../../NSImageAdditions.h"
 #import "../../ProjectDocument.h"
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' projectmanager.app-0.2~/ProjectManager-0.2/ProjectTypes/Application/ApplicationTypeAttributes.m projectmanager.app-0.2/ProjectManager-0.2/ProjectTypes/Application/ApplicationTypeAttributes.m
--- projectmanager.app-0.2~/ProjectManager-0.2/ProjectTypes/Application/ApplicationTypeAttributes.m	2006-07-09 17:11:43.000000000 +0300
+++ projectmanager.app-0.2/ProjectManager-0.2/ProjectTypes/Application/ApplicationTypeAttributes.m	2010-05-21 12:26:59.000000000 +0300
@@ -23,12 +23,7 @@
 
 #import "ApplicationTypeAttributes.h"
 
-#import <Foundation/NSString.h>
-
-#import <AppKit/NSNibLoading.h>
-#import <AppKit/NSTextField.h>
-#import <AppKit/NSTableView.h>
-#import <AppKit/NSImageView.h>
+#import <AppKit/AppKit.h>
 
 #import "../../ProjectDocument.h"
 
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' projectmanager.app-0.2~/ProjectManager-0.2/ProjectWindowController.m projectmanager.app-0.2/ProjectManager-0.2/ProjectWindowController.m
--- projectmanager.app-0.2~/ProjectManager-0.2/ProjectWindowController.m	2010-05-21 12:25:06.000000000 +0300
+++ projectmanager.app-0.2/ProjectManager-0.2/ProjectWindowController.m	2010-05-21 12:25:18.000000000 +0300
@@ -23,20 +23,7 @@
 
 #import "ProjectWindowController.h"
 
-#import <AppKit/NSImage.h>
-#import <AppKit/NSWindow.h>
-#import <AppKit/NSWindow+Toolbar.h>
-#import <AppKit/NSToolbar.h>
-#import <AppKit/NSToolbarItem.h>
-#import <AppKit/NSBox.h>
-#import <AppKit/NSPopUpButton.h>
-#import <AppKit/NSScrollView.h>
-#import <AppKit/NSTextView.h>
-#import <AppKit/NSTextStorage.h>
-
-#import <Foundation/NSBundle.h>
-#import <Foundation/NSException.h>
-#import <Foundation/NSNotification.h>
+#import <AppKit/AppKit.h>
 
 #import "ProjectDocument.h"
 #import "ProjectModule.h"
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' projectmanager.app-0.2~/ProjectManager-0.2/SourceEditorDocument.m projectmanager.app-0.2/ProjectManager-0.2/SourceEditorDocument.m
--- projectmanager.app-0.2~/ProjectManager-0.2/SourceEditorDocument.m	2010-05-21 12:25:17.000000000 +0300
+++ projectmanager.app-0.2/ProjectManager-0.2/SourceEditorDocument.m	2010-05-21 12:25:18.000000000 +0300
@@ -23,25 +23,7 @@
 
 #import "SourceEditorDocument.h"
 
-#import <Foundation/NSKeyedArchiver.h>
-#import <Foundation/NSBundle.h>
-#import <Foundation/NSFileHandle.h>
-#import <Foundation/NSIndexSet.h>
-#import <Foundation/NSKeyValueCoding.h>
-#import <Foundation/NSString.h>
-#import <Foundation/NSTask.h>
-#import <Foundation/NSUserDefaults.h>
-#import <Foundation/NSValue.h>
-
-#import <AppKit/NSFont.h>
-#import <AppKit/NSImage.h>
-#import <AppKit/NSLayoutManager.h>
-#import <AppKit/NSPanel.h>
-#import <AppKit/NSParagraphStyle.h>
-#import <AppKit/NSRulerView.h>
-#import <AppKit/NSScrollView.h>
-#import <AppKit/NSTextStorage.h>
-#import <AppKit/NSTextView.h>
+#import <AppKit/AppKit.h>
 
 #import <../HighlighterKit-0.1/HighlighterKit.h>
 
